namespaces-disallowlist.yaml

```diff
@@ -16,3 +16,9 @@
 - org_mozilla_tiktokreporter
 - moso_*
 - regrets_reporter
+- mozphab
+- mozregression
+- pine
+- treeherder
+- sync
+- mozillavpn_backend_cirrus
```

Comment:
`mozillavpn_backend_cirrus` was only recently added ([bug 1867770](https://bugzilla.mozilla.org/show_bug.cgi?id=1867770)) and just hasn't started being used yet.

@brizental are there still plans to use Cirrus with VPN?

Reply:
Nope, that whole effort was frozen after the layoffs. We might pick it up agai nin the future, but that is unclear at the moment. If this can be reverted easily in the future, it's fine by me.